README.MD

```diff
@@ -39,27 +39,31 @@ npm install --save-dev @beyonk/gdpr-cookie-consent-banner
 then import the banner into your code:
 
 ```js
-  import attachBanner from '@beyonk/gdpr-cookie-consent-banner'
-  attachBanner(document.body)
+import attachBanner from '@beyonk/gdpr-cookie-consent-banner'
+import '@beyonk/gdpr-cookie-consent-banner/dist/style.css' // import optional styles
+
+attachBanner(document.body)
 ```
 
 #### Using raw javascript
 
 ##### From a CDN
 
 ```html
-<script src="//unpkg.com/@beyonk/gdpr-cookie-consent-banner@3.1.0/dist/browser/bundle.min.js"></script>
+<link rel="stylesheet" href="//unpkg.com/@beyonk/gdpr-cookie-consent-banner@5.1.0/dist/style.css">
```

Comment:
@antony I assumed you will bump the version to 5.1.0 after my PR has been merged, so I updated both CDN links to reflect that. I hope that's OK with you.